src/include/misc.i

```diff
@@ -70,3 +70,19 @@ __wt_verbose(WT_SESSION_IMPL *session, int flag, const char *fmt, ...)
 	return (0);
 #endif
 }
+
+/*
+ * __wt_diagnostic_yield --
```

Comment:
@sueloverso
> I'd also like to have a diagnostic function to introduce aborts or errors in some code paths.

That could be a separate ticket? I'd like to get this pushed and into the stress tests.